### ISSUE TITLE
Logging: Understanding Head-to-Wall space

### ISSUE THREAD

**Issue by bastianilso** [med-material/Whack_A_Mole_VR#125]
* Make it easier for the therapist to align the patient with the x-axis, display head position/rotation in therapist UI
* Show a button in the UI from which the therapist can set what is the equivalent of "looking straight ahead on the wall". Log what the current head-rotation, head-position and controller-positions are, to give us an idea of what are the defaults.
* Log CD Gain (multiplier / translation) from Motor spaces

**Comment by bastianilso** [comment]
Button in UI is now implemented since #128.
Remaining todo:
- Log CD Gain in motor spaces

**Comment by bastianilso** [comment]
@hendrikknoche regarding logging the CD gain, could you give me a quick refresh on how we best should do this? is there a formula I would need to implement in programming?

**Comment by hendrikknoche** [comment]
assuming that we 'stretch' input from X and Y (and not Z, right?) independently we should log the scaling factor you're using as a result of the motor space calibration. I assume that we would usually use a 1:1 mapping.

**Comment by bastianilso** [comment]
The code looks like so

```
        float normalizedX = (coord.x - motorSpaceTopLeft.x) / (motorSpaceTopRight.x - motorSpaceTopLeft.x);
        float wallX = ((wallSpaceTopRight.x - wallSpaceTopLeft.x) * normalizedX) + wallSpaceTopLeft.x;
```
and is run on every pointer update - so i currently dont store a scaling factor per se.
but it could be something I can derive from the conversion above (and log)
ie. if we assume the wall and the motorspace start in the same position then:

`scalingFactor = (wallSpaceTopRight.x - wallSpaceTopLeft.x)  / (motorSpaceTopRight.x - motorSpaceTopLeft.x);`

**Comment by bastianilso** [comment]
implemented in commit 7465588
